js/wee.history.js

```diff
@@ -344,17 +372,11 @@
 			var a = W._doc.createElement('a');
 			a.href = request.root + request.url;
 
-			if (! support || ! _isValid(a)) {
+			if ((! support || ! _isValid(a)) && ! conf.force) {
```

Comment:
CORS-based external requests, I used it when pulling in JSON data via a subdomain.